### PR TITLE
[Toolbar] set selection initially in fixed controller and toolbar...

### LINKED ISSUE #1991: [Fixed Position] Add button is missing when new object is created
When a new fixed position display is created the Add button is missing in the toolbar. Clicking anywhere on the canvas makes it appear in the toolbar.

### PR DESCRIPTION
**Pull request by psarram**
... to make the add button appear in the toolbar when a fixed position is created.

Remove selection change listener on destroy.

Start a digest cycle when handling selection in toolbar to avoid delays in toolbar.

Fixes #1991, #1987

### CODE REVIEW

File: platform/commonUI/edit/src/representers/EditToolbarRepresenter.js [hunk 1]
```diff
@@ -130,26 +125,23 @@ define(
...
-            // Expose the toolbar object to the parent scope
-            initialize(definition);
-            // Create a selection scope
-            this.setSelection(new EditToolbarSelection(this.openmct));
+            // Add toolbar selection to scope.
+            this.scope.selection = new EditToolbarSelection(
+                this.scope,
+                this.openmct
+            );
             // Initialize toolbar to an empty selection
```
Comment (larkin) [inline platform/commonUI/edit/src/representers/EditToolbarRepresenter.js]: should update, "initialize toolbar to current selection"

**Comment by psarram** [comment]
@larkin I fixed the tests and updated the comment.

**Comment by psarram** [comment]
**Author Checklist**

1. Changes address original issue? Y
2. Unit tests included and/or updated with changes? Y
3. Command line build passes? Y
4. Changes have been smoke-tested? Y

**Comment by larkin** [comment]
# Reviewer Checklist

1. Changes appear to address issue? Y
2. Appropriate unit tests included? Y
3. Code style and in-line documentation are appropriate? Y
4. Commit messages meet standards? Y

**Pull request merged**